### PR TITLE
feat: able to customize worker' images

### PR DESCRIPTION
**Pull request by 0xff-dev**
#### What type of PR is this?
/kind feature

#### What this PR does / why we need it
feat: able to customize worker' images

#### Which issue(s) this PR fixes

Fix https://github.com/kubeagi/arcadia/issues/545

### CODE REVIEW

File: pkg/worker/loader.go [hunk 1]
```diff
@@ -100,8 +102,8 @@ func (loader *LoaderOSS) Build(ctx context.Context, model *arcadiav1alpha1.Typed
 
 	container := &corev1.Container{
 		Name:            "loader",
-		Image:           "kubeagi/minio-mc:RELEASE.2023-01-28T20-29-38Z",
-		ImagePullPolicy: "IfNotPresent",
+		Image:           loader.worker.Spec.Loader.Image,
```
Comment (bjwswang) [inline pkg/worker/loader.go:105]: We should use `kubeagi/minio-mc:RELEASE.2023-01-28T20-29-38Z` as the default image. And once user provided a different one,we change it.

File: pkg/worker/loader.go [hunk 2]
```diff
@@ -157,8 +160,8 @@ func (r *RDMALoader) Build(ctx context.Context, _ *arcadiav1alpha1.TypedObjectRe
 
 	container := &corev1.Container{
 		Name:            "rdma-loader",
-		Image:           "wetman2023/floo:23.12",
-		ImagePullPolicy: "IfNotPresent",
+		Image:           r.worker.Spec.Loader.Image,
```
Comment (bjwswang) [inline pkg/worker/loader.go:163]: same as above

File: pkg/worker/loader.go (hunk 1 above)
Comment (bjwswang) [reply pkg/worker/loader.go:105]: Keep the default image names

File: pkg/worker/loader.go [hunk 3]
```diff
@@ -29,6 +29,11 @@ import (
 	"github.com/kubeagi/arcadia/pkg/datasource"
 )
 
+const (
+	defaultLoaderImage     = "kubeagi/minio-mc:RELEASE.2023-01-28T20-29-38Z"
```
Comment (bjwswang) [inline pkg/worker/loader.go:33]: defaultOSSLoaderImage

File: pkg/worker/loader.go [hunk 4]
```diff
@@ -98,10 +105,14 @@ func (loader *LoaderOSS) Build(ctx context.Context, model *arcadiav1alpha1.Typed
 		url = loader.endpoint.SchemeInternalURL()
 	}
 
+	img := defaultLoaderImage
+	if loader.worker.Spec.Loader.Image != "" {
+		img = loader.worker.Spec.Loader.Image
+	}
```
Comment (bjwswang) [inline pkg/worker/loader.go:111]: set default pull policy too

**Comment by Abirdcfly** [comment]
Chart version also need update？

File: pkg/worker/loader.go (hunk 4 above)
Comment (0xff-dev) [reply pkg/worker/loader.go:111]: https://github.com/kubeagi/arcadia/pull/561/files#diff-8f0e3fc5aa037de327c89f467cafeabaf458c3026d56305ea04b3a7b5feb9c9fR30

Default pull policy is `IfNotPresent`

**Review by bjwswang — APPROVED** [review]

File: pkg/worker/runner.go [hunk 5]
```diff
@@ -30,6 +30,11 @@ import (
 	"github.com/kubeagi/arcadia/pkg/config"
 )
 
+const (
+	defaultFastChatImage    = "kubeagi/arcadia-fastchat-worker:v0.2.0"
+	defaultFastChatLLMImage = "kubeagi/arcadia-fastchat-worker:vllm-v0.2.0"
```
Comment (bjwswang) [inline pkg/worker/runner.go:35]: defaultFastchatVLLMImage

**Review by bjwswang — CHANGES_REQUESTED** [review]

**Review by bjwswang — APPROVED** [review]

**Pull request merged**